instrumentation/opentelemetry-instrumentation-asgi/tests/test_asgi_middleware.py

```diff
@@ -1692,21 +1683,12 @@ def test_query_string(self):
             attrs[SpanAttributes.HTTP_URL], "http://127.0.0.1/?foo=bar"
         )
 
-    def test_query_string_new_semconv(self):
-        self.scope["query_string"] = b"foo=bar"
-        attrs = otel_asgi.collect_request_attributes(
-            self.scope,
-            _HTTPStabilityMode.HTTP,
-        )
-        self.assertEqual(attrs[URL_FULL], "http://127.0.0.1/?foo=bar")
-
     def test_query_string_both_semconv(self):
         self.scope["query_string"] = b"foo=bar"
         attrs = otel_asgi.collect_request_attributes(
             self.scope,
             _HTTPStabilityMode.HTTP_DUP,
         )
-        self.assertEqual(attrs[URL_FULL], "http://127.0.0.1/?foo=bar")
```

Comment:
Instead of only removing can we assert the scheme, server address and path?

Reply:
Thanks for your review @emdneto. All changes are included in my new commit.